### ISSUE TITLE
Add PR template to this GitHub repository

### ISSUE THREAD

**Issue by chingucoding** [microsoft/microsoft-ui-xaml#1212]
Over at [Xaml Controls Gallery](https://github.com/microsoft/xaml-controls-gallery), when creating a PR there is a template available, which one can fill out. I found that extremely helpful when creating PRs and it probably makes it easier to work on other PRs.

Can we introduce PR templates in this repository too? 😄

**Comment by jevansaks** [comment]
Reasonable request. Do you have parts of the reminder checklist in that repo (or other repos) that you have found useful?

**Comment by chingucoding** [comment]
I definitely think think that a quick overview of the changes, a more detailed list of changes and a motivation (e.g. an issue) help a lot. 
When dealing with style related PR's screenshots could be useful, but that is a category that does not apply every time. 
Testing methodology also can sometimes be useful in my opinion as one can point out what they have done in order to validate that the changes are working.

So in summary I think these things are really helpful:
* Quick summary
* Motivation
* Screenshot (if applicable)

and these are nice to have but:
* More detailed list of changes
* How was this tested/testing methodology
